### DISCUSSION TITLE
CVSSv3 support

### DISCUSSION THREAD

**Discussion by tpaiii3** [immauss/openvas#231] [Q&A] [answered]
Is there any way to use the CVSSv3 scoring system instead of CVSSv2?

now, I'm on Greenbone Security Assistant Version 22.5.3

**Comment by immauss** [comment, marked as answer]
Sorry for the delayed answer, response. 

First, make sure you are running the latest versions. 

In mine, I see CVSS3 scoring in the latest NVTs. I believe this is dependent on the individual NVT and more on its age than anything else. Certainly the latest NVTs seem to all be using v3.  Either way, this unfortunately outside my ability to change. 

-Scott
